modules/keyvault/main.tf

```diff
@@ -0,0 +1,90 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "keyvault" {
+  name                        = "tikweb-keyvault-${var.env_name}"
+  location                    = var.resource_group_location
+  resource_group_name         = var.resource_group_name
+  enabled_for_disk_encryption = true
+  tenant_id                   = data.azurerm_client_config.current.tenant_id
+  soft_delete_retention_days  = 7
+  purge_protection_enabled    = false
```

Comment:
Do we want more protection against mistakes? `prevent_destroy` might also make sense.